build/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
```

Comment:
This file isn't needed, and can be deleted.

Reply:
Done